Ktisis/Interface/Windows/ActorEdit/EditEquip.cs

```diff
@@ -41,9 +52,12 @@ public class EditEquip {
 		// UI Code
 
 		public unsafe static void Draw() {
+			DrawControls();
```

Comment:
Move this down below the initialization of `Items`…

Reply:
It should be done with 87e4845.